styles/Card.module.css

```diff
@@ -0,0 +1,45 @@
+.card_list {
+  width: 530px;
+  display: grid;
+  grid-template-rows: auto auto;
```

Comment:
I think it is used on the card component and doesn't see any necessary to use a grid box.

Reply:
Replaced with flex